tests/components/google_mail/test_sensor.py

```diff
@@ -29,15 +40,15 @@ async def test_sensors(hass: HomeAssistant, setup_integration: ComponentSetup) -
         "httplib2.Http.request",
         return_value=(
             Response({}),
-            bytes(load_fixture("google_mail/get_vacation_off.json"), encoding="UTF-8"),
+            bytes(load_fixture(f"google_mail/{fixture}.json"), encoding="UTF-8"),
```

Comment:
Pass the integration domain name as a parameter instead:
https://github.com/home-assistant/core/blob/193b2694a98cb8a8dfa2c414838e156bc79def5a/tests/common.py#L437-L437